### PR TITLE
Adds EngiCart

### PR DESCRIPTION
**Pull request by ghost**
http://i.imgur.com/18w9fkq.png Can hold three toolboxes, metal, plasteel, glass, tape, and a flashlight.

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
If you want to go ahead and add in JaniCarts and Holosigns, feel free! We lack both.

**Comment by ghost** [comment]
@Fox-McCloud 
Done. Oh, I didn't see the holosigns part. I'll add them in maybe later. Kinda burnt out being my first time of really coding.

File: code/game/objects/structures/engicart.dm [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+	if(myflashlight)
+		dat += "<a href='?src=\ref[src];flashlight=1'>[myflashlight.name]</a><br>"
+	if(mybluetoolbox)
+		dat += "<a href='?src=\ref[src];bluetoolbox=1'>[mybluetoolbox.name]</a><br>"
+	if(myredtoolbox)
+		dat += "<a href='?src=\ref[src];redtoolbox=1'>[myredtoolbox.name]</a><br>"
+	if(myyellowtoolbox)
+		dat += "<a href='?src=\ref[src];yellowtoolbox=1'>[myyellowtoolbox.name]</a><br>"
+	if(myengitape)
+		dat += "<a href='?src=\ref[src];myengitape=1'>[myengitape.name]</a><br>"
```
Comment (Glloyd) [inline code/game/objects/structures/engicart.dm:103]: Note that this means you won't be able to remove the tape from the cart. Just remove the my and you'll be good.

Comment (ghost) [inline code/game/objects/structures/engicart.dm:103]: Oh god, how'd I miss that.

**Pull request merged**